internal/config/types/schema.go

```diff
@@ -137,12 +137,14 @@ type NodeUser struct {
 }
 
 type Partition struct {
-	GUID     string `json:"guid,omitempty"`
-	Label    string `json:"label,omitempty"`
-	Number   int    `json:"number,omitempty"`
-	Size     int    `json:"size,omitempty"`
-	Start    int    `json:"start,omitempty"`
-	TypeGUID string `json:"typeGuid,omitempty"`
+	GUID               string  `json:"guid,omitempty"`
+	Label              *string `json:"label,omitempty"`
+	Number             int     `json:"number,omitempty"`
```

Comment:
It looks like this (and most of the other `int`s) is really a `uint`. Does the schematyper support specifying that?